stdlib/date/date.flux

```diff
@@ -1,397 +1,463 @@
 // Package date provides date and time constants and functions.
+//
+// introduced: 0.37.0
+// tags: date/time
 package date
 
 
-// second is a function that returns the second of a specified time. Results
-//  range from [0 - 59].
+// second returns the second of a specified time. Results range from `[0 - 59]`.
 //
 // ## Parameters
-// - `t` is the time to operate on.
+// - t: Time to operate on.
 //
-//   Use an absolute time, relative duration, or integer. durations are
-//   relative to `now()`.
+//   Use an absolute time, relative duration, or integer.
+//   Durations are relative to `now()`.
 //
-// ## Return the second of a time value
+// ## Examples
+// 
+// ### Return the second of a time value
 //
-// ```
+// ```no_run
```

Comment:
Why can't these be run?

Reply:
I don't think it's a matter of "can't be run." I think it's that they don't need to be run. I assumed that we'd run examples to generate and capture input and output streams. Where these don't have any streams of tables, they don't need to be run. But that assumption may be incorrect.

@nathanielc Should I have examples with no input/output tables still run?

Reply:
If an example doesn't have any input or output then we won't capture and data but they can still run. If they get an error that error will be reported. Therefore we don't need to mark them as `no_run` unless we expect the code will error or have another reason for not running.

Reply:
If you do get an error trying to run any of these examples that you don't expect ping me and I'll take a look. There could easily be a bug in the example runner at this stage in the process.

Reply:
After writing this comment I thought of a case were an error would not be reported so I fixed it here https://github.com/influxdata/flux/pull/4215